pyOCD/target/target_k20d50m.py

```diff
@@ -30,4 +31,5 @@ class K20D50M(Kinetis):
     def __init__(self, link):
         super(K20D50M, self).__init__(link, self.memoryMap)
         self.mdm_idr = 0x001c0000
+        self._svd_location = SVDFile(vendor="Freescale", filename="MK20D5.xml", is_local=False)
```

Comment:
Do the svd files need to be copied locally to be used?  What can be done with the svd files?  Can they be easily be used from within python, or are they only useful for the pyocd-tool?